.gitignore

```diff
@@ -97,3 +97,6 @@ network_closure.sh
 
 # Downloaded kubernetes binary release tar ball
 kubernetes.tar.gz
+
+# Generated directories in the source
```

Comment:
super minor nit - this is a file, not a directory.

also, may be worth saying this is generated in the build using `go generate` and explicitly not checked in, to differentiate it from the code we *do* check in.